nomad/local/grapl-local-infra.nomad

```diff
@@ -555,11 +554,52 @@ job "grapl-local-infra" {
 
       config {
         image = "scylladb-ext:${var.image_tag}"
+        args = [
+          # Set up scylla in single-node mode instead of in overprovisioned mode, ie DON'T use all available cpu/memory
+          "--smp", "1"
+        ]
         ports = ["internal_node_rpc_1", "internal_node_rpc_2", "cql", "thrift", "rest"]
+
+        # Configure a data volume for scylla. See the "Configuring data volume for storage" section in
+        # https://hub.docker.com/r/scylladb/scylla/
+        mount {
+          type     = "volume"
+          target   = "/var/lib/scylla"
+          source   = "scylla-data"
+          readonly = false
+          volume_options {
+            # Upon initial creation of this volume, *do* copy in the current
+            # contents in the Docker image.
+            no_copy = false
+            labels {
+              maintainer = "Scylla"
+            }
+          }
+        }
+
       }
 
       service {
         name = "scylla"
+
+        check {
+          type = "script"
+          name = "nodestatus_check"
+          # We use bin/bash so we can pipe to grep
+          command  = "bin/bash"
+          args     = ["nodetool", "status", "|", "grep", "'UN'"]
+          interval = "30s"
+          timeout  = "10s"
+
+          check_restart {
+            # Wait up to 4 minutes for scylla to be ready, ie this is a readiness check
+            grace           = "4m"
```

Comment:
4m seems high, would suggest 1m

Reply:
Yep, this is a holdover from prior to switching to single node mode